src/content/guides/deploying-istio/index.md

```diff
@@ -0,0 +1,332 @@
+
+### AWS verification
+
+Get the load balancer hostname. 
+
+```nohighlight
+$ kubectl get service istio-ingressgateway -n istio-system -o jsonpath="{.status.loadBalancer.ingress[0].hostname}"
+```
+
+This will return the URL under which the deployed app should reply. As we have set wildcard `*` in the hostname of the virtual service all `/healthz` traffic will be forwarded to the service.
+
+Executing this curl command should give you a `200` response.
+
+```nohighlight
+$ curl http://<ELB_HOSTNAME>/healthz -I
+```
+
+### Azure verification
+
+Get the ingress controller IP assigned by Azure.
+
+```nohighlight
+$ kubectl get svc istio-ingressgateway -n istio-system -o jsonpath="{.status.loadBalancer.ingress[0].ip}"
+```
+
+And it will return the URL which the deployed app should reply to. Same as AWS, the wildcard `*` set as the hostname in the virtual service will send all traffic from `/healthz` path to the liveness service.
+
+Running this curl command should give you a `200` response.
+
+```nohighlight
+$ curl http://<IP>/healthz -I
+```
+
+## On premise (KVM)
+
+As you may probably know, in our [`on premise installations`](https://docs.giantswarm.io/basics/onprem-architecture/) the tenant pods live inside the control plane pods. It means Giant Swarm creates a forwarding between the external ingress service to the ingress placed inside the tenant cluster. By default, the tenant clusters has an nginx ingress controller allocated out of the box. But, in case you want to use Istio ingress controller you need to change the current ingress controller (remember it will affect all current traffic for that tenant cluster). Another option is asking our team to allocate a new redirection from the parent endpoint to the Istio controller. With the latter, you will have the two ingress controllers exposed to Internet.
+
+After obtaining the ports, modify the ingress gateway to set the correct configuration.
+
+```nohighlight
+$ kubectl edit service istio-ingressgateway -n istio-system
+```
+
+```yaml
+...
+  - name: http2
+    nodePort: <PORT_PROVIDED_BY_GS_OR_DEFAULT_ONE>
+    port: 80
+    protocol: TCP
+    targetPort: 80
+  - name: https
+    nodePort: <PORT_PROVIDED_BY_GS_OR_DEFAULT_ONE>
+    port: 443
+    protocol: TCP
+    targetPort: 443
+...
+```
+
+After configured it, running the next command you should get a `200` response.
+
+```nohighlight
+$ curl http://<LOAD_BALANCER_IP>:<PORT_TENANT_CLUSTER>/healthz -I
+```
+
+Giant Swarm encourages you to leave this liveness app up and running in order to make our monitoring happy. By default, nginx ingress controller provides a `/healthz` endpoint to check ICs are up, but the Istio controller lacks of this functionality. For that reason, the liveness app will give us the health signal required to have the correct observability of your istio clusters.
```

Comment:
I guess this is obsolete if we do not support replacement of the IC for now. We can think of adding this once we can actually make IC optional